src/loadAndResolveLessVars.js

```diff
@@ -17,7 +16,7 @@ export function loadLessWithImports(entry) {
     const importPath = match[1]
     const fullImportPath = /\.less$/.test(importPath) ? importPath : `${importPath}.less`
     const resolvedImportPath = /^~/.test(importPath)
-      ? resolve(root, 'node_modules', fullImportPath.slice(1))
+      ? enhancedResolve.sync(__dirname, fullImportPath.slice(1))
```

Comment:
1. Could you, please, explain a bit about the need for using `enhanced-resolve`?
1. *[!]* Using `__dirname` starts the lookup at the location of _this file_. The intent is to do lookup at the place of script execution.

Reply:
Thanks for your reply !
1. ```enhanced-resolve``` is a package used by webpack for file resolving, I think it does a recursion search for the given file
2. Emm... I am used to using __dirname as the start path, maybe ```resolve("./")``` also works

Reply:
1. Ok, makes sense.
1. There's a crucial difference, see https://stackoverflow.com/questions/8131344/what-is-the-difference-between-dirname-and-in-node-js .

Reply:
Less dependencies should IMO be resolved from project directory, not from the place where _this file_ (`loadAndResolveLessVars.js`) is located. (That's what `__dirname` means - directory in which the piece of code is located.)